cranelift/codegen/meta/src/isa/x86/registers.rs

```diff
@@ -4,7 +4,7 @@ pub(crate) fn define() -> IsaRegs {
     let mut regs = IsaRegsBuilder::new();
 
     let builder = RegBankBuilder::new("FloatRegs", "xmm")
-        .units(32)
+        .units(16)
```

Comment:
FIXME set back to 32 when FPR32 is re-added

Reply:
I don't think I need to remind myself where to expand the register bank...